assets/src/edit-story/app/api/apiProvider.js

```diff
@@ -271,6 +271,17 @@ function APIProvider({ children }) {
     });
   }, [currentUser]);
 
+  const updateCurrentUser = useCallback(
```

Comment:
If you're gonna add some karma tests for quick tips, might need to mock this in `assets/src/edit-story/karma/fixture/fixture.js`

Reply:
I'll at least create a spy here